code/frameworks/svelte-vite/src/preset.ts

```diff
@@ -12,8 +13,23 @@ export const viteFinal: StorybookConfig['viteFinal'] = async (config, { presets
 
   plugins.push(svelteDocgen(config));
 
+  removeSvelteKitPlugin(plugins);
+
   return {
     ...config,
     plugins,
   };
 };
+
+const removeSvelteKitPlugin = (plugins: PluginOption[]) => {
```

Comment:
I'd still love to find a way to avoid having to do this, since we'll start getting complaints about sveltekit aliases not working (even though there seems to be some debate over whether they should be used at all inside components).  But, this seems like a reasonable workaround for now, until we can figure it out.

Reply:
I agree, let's continue investigating the core issue and do a follow up to this.